src/backend/commands/matview.c

```diff
@@ -510,6 +520,52 @@ ExecRefreshMatView(RefreshMatViewStmt *stmt, const char *queryString,
 		relpersistence = matviewRel->rd_rel->relpersistence;
 	}
 
+	/* delete immv triggers */
+	if (matviewRel->rd_rel->relisivm)
+	{
```

Comment:
You can use RelationIsIVM macro here, again.

Why triggers should be deleted even when skipData is not true?

Reply:
> You can use RelationIsIVM macro here, again.
All right.

> Why triggers should be deleted even when skipData is not true?
I think that this logic is simple for condition of calling CreateIvmTriggerOnBaseTables().
But its process is unnecessary. So I will fix condition of create/delete trigger.